opensearch/config/opensearch.yml

```diff
@@ -0,0 +1,10 @@
+## Cluster Settings
+cluster.name: stac-cluster
+node.name: es01
+network.host: 0.0.0.0
```

Comment:
We may want to make this `127.0.0.1` so that the service isn't available to everyone else in the coffee shop.

Reply:
Looking at how docker-compose is using this, I think it should be fine.